static/js/copy_and_paste.js

```diff
@@ -89,18 +103,33 @@ function copy_handler() {
             skip_same_td_check = true;
         }
 
+        more_than_two_blocks = check_blocks(start_id,end_id);
```

Comment:
Maybe we can name this "multiple_messages_selected"?

Reply:
also, I think there is a lint error on this line, due to not having a space after the comma?